hudi-spark/src/main/scala/org/apache/hudi/AvroConversionHelper.scala

```diff
@@ -338,12 +337,13 @@ object AvroConversionHelper {
         }
       case structType: StructType =>
         val schema: Schema = SchemaConverters.toAvroType(structType, nullable = false, structName, recordNamespace)
+        val childNameSpace = if (recordNamespace != "") s"$recordNamespace.$structName" else structName
```

Comment:
nit: recordNameSpace.size() == 0 ?

Reply:
This line is directly copied from: https://github.com/apache/spark/blob/master/external/avro/src/main/scala/org/apache/spark/sql/avro/SchemaConverters.scala#L175
Do you still want me to change it?

Reply:
I mean its a nit.. not sure if the line being in spark would change how feel about it.. but again its a nit, happy to let this be this way :)